pkg/operator/ceph/object/controller.go

```diff
@@ -408,6 +415,12 @@ func (r *ReconcileCephObjectStore) reconcileCreateObjectStore(cephObjectStore *c
 		if err != nil {
 			return reconcile.Result{}, errors.Wrapf(err, "failed to create object store %q", cephObjectStore.Name)
 		}
+
+		// Reconcile rgw admin ops api user
+		err = r.reconcileRGWAdminOpsUser(cephObjectStore)
```

Comment:
Is this reconcileRGWAdminOpsUser() the same as above on lines 363-367? Looks like it's the same call for external and local clusters. Could this be factored out in common? In other words, move it out of the if/else block?

Reply:
I can try.